webapp2.py

```diff
@@ -119,7 +123,7 @@
     505: 'HTTP Version not supported',
 }
 status_reasons.update(_webapp_status_reasons)
-for code, message in _webapp_status_reasons.iteritems():
+for code, message in _webapp_status_reasons.items():
```

Comment:
How about these other instances? We need to be sure (or write a helper like `six.iteritems`)